addons/token-exchange/blue_secret_controller.go

```diff
@@ -125,6 +140,51 @@ func (c *blueSecretTokenExchangeAgentController) sync(ctx context.Context, syncC
 	return nil
 }
 
+func (c *blueSecretTokenExchangeAgentController) enableMirroring(storageClusterName string) error {
+	ctx := context.TODO()
+	cl, err := getClient(c.spokeKubeConfig)
+	if err != nil {
+		return err
+	}
+	var sc ocsv1.StorageCluster
+	err = cl.Get(ctx, types.NamespacedName{
+		Name:      storageClusterName,
+		Namespace: StorageNamespace,
+	}, &sc)
+
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return fmt.Errorf("could not find storagecluster %q in namespace %v: %v", storageClusterName, StorageNamespace, err)
+		}
+		return err
+	}
+	sc.Spec.Mirroring.Enabled = true
+	err = cl.Update(ctx, &sc)
+	return err
+}
+func (c *blueSecretTokenExchangeAgentController) enableCSIOmapGenerator() error {
+	cl, err := getClient(c.spokeKubeConfig)
+	if err != nil {
+		return err
+	}
+	var rcm corev1.ConfigMap
+	err = cl.Get(context.TODO(), types.NamespacedName{
+		Name:      RookConfigMapName,
+		Namespace: StorageNamespace,
+	}, &rcm)
+
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return fmt.Errorf("could not find rook-ceph-config-map: %v", err)
+		}
+		return err
+	}
+
+	rcm.Data[RookCSIEnableKey] = "true"
```

Comment:
We need to set `CSI_ENABLE_VOLUME_REPLICATION` and `CSI_ENABLE_OMAP_GENERATOR` to true.

Reply:
done